cluster/addons/istio/auth/istio-auth.yaml

```diff
@@ -3,13 +3,15 @@ kind: Namespace
 metadata:
   name: kube-public
   labels:
+    addonmanager.kubernetes.io/mode: Reconcile
```

Comment:
Most (all?) of the other addon files that include this label also include

`kubernetes.io/cluster-service: "true"`

above it. Can you add that label as well for consistency?

Reply:
Maybe it's just cargo-culting that has caused it to show up in other places then. e.g.

https://github.com/kubernetes/kubernetes/blob/3e694c38e0b423b1c79a0d7c69f74137a357653f/cluster/addons/storage-class/openstack/default.yaml#L8

or

https://github.com/kubernetes/kubernetes/blob/1bc5f7cfa335e2198fee945a6e9f83e780ee41f4/cluster/addons/calico-policy-controller/typha-horizontal-autoscaler-clusterrole.yaml#L6

neither of which are services. 

And according to [this comment](https://github.com/kubernetes/kubernetes/blob/b5f61ac129019d314e473584c1491b7ca62144c7/cluster/addons/addon-manager/kube-addons.sh#L41-L45) the label is deprecated so you probably shouldn't add it. Sorry for the distraction.